packages/teleport/src/Nodes/AddNode/Iam/Iam.tsx

```diff
@@ -0,0 +1,267 @@
+              .
+            </Text>
+            <Box mb={4}>
+              <Text bold as="span">
+                Step 1
+              </Text>{' '}
+              - Download Teleport package to your computer
+              <DownloadLinks isEnterprise={isEnterprise} version={version} />
+            </Box>
+            <Box>
+              <Text bold as="span">
+                Step 2
+              </Text>{' '}
+              - Setup AWS Rules
+              {rules.map((rule, index) => (
+                <RuleBox
+                  key={index}
+                  bg="primary.lighter"
+                  borderRadius="2"
+                  mt={1}
+                  p="3"
+                  position="relative"
+                >
+                  <Text typography="h5">{`Rule #${index + 1}`}</Text>
+                  {rule.isCollapsed && (
+                    <ButtonEditRule
+                      title="edit rule"
+                      onClick={() =>
+                        setRuleAtIndex(index, { ...rule, isCollapsed: false })
+                      }
+                    >
+                      <Pencil />
+                    </ButtonEditRule>
+                  )}
+                  {!rule.isCollapsed && (
+                    <>
+                      {index !== 0 && (
+                        <ButtonRemoveRule onClick={() => handleRemove(index)}>
+                          Remove
+                        </ButtonRemoveRule>
+                      )}
+                      <Box>
+                        <FieldInput
+                          label="AWS Account"
+                          autoFocus
+                          onChange={e =>
+                            setRuleAtIndex(index, {
+                              ...rules[index],
+                              awsAccount: e.target.value,
+                            })
+                          }
+                          rule={value => requiredAwsAccount(value, rule)}
+                          placeholder="111111111111"
+                          value={rule.awsAccount}
+                        />
+                      </Box>
+                      <FieldInput
+                        mb={2}
+                        label="AWS ARN"
+                        onChange={e =>
+                          setRuleAtIndex(index, {
+                            ...rules[index],
+                            awsArn: e.target.value,
+                          })
+                        }
+                        placeholder="arn:aws:sts::111111111111:assumed-role/teleport-node-role/i-*"
+                        value={rule.awsArn}
+                      />
+                    </>
+                  )}
+                </RuleBox>
+              ))}
+            </Box>
+            <Box
+              onClick={() =>
+                setRules([
+                  ...rules,
+                  { awsAccount: '', awsArn: '', isCollapsed: false },
+                ])
+              }
```

Comment:
i think it will be cleaner and leaner if we add a helper func `addRule` then:
```suggestion
              onClick={addRule}
```